.travis.yml

```
@@ -0,0 +1,23 @@
+language: java
+
+# Setting sudo to false will cause Travis to use Containers.
+# To use Docker's privileged mode, we need to enable sudo.
+sudo: required
+
+matrix:
+  include:
+    - jdk: openjdk11
+
+addons:
+  sonarcloud:
+    organization: exasol
+
+install:
+    - travis_retry mvn dependency:resolve
+
+script:
+  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent org.jacoco:jacoco-maven-plugin:prepare-agent-integration verify sonar:sonar
```

the SONAR_TOKEN env variable is missing
```suggestion
    - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent org.jacoco:jacoco-maven-plugin:prepare-agent-integration verify sonar:sonar -Dsonar.login=${SONAR_TOKEN}
```

---

This is not necessary I think, Sonar works fine without it, it a default value